test/unittests/rest/RestService_test.cpp

```diff
@@ -136,7 +136,7 @@ TEST(RestService, payloadParse)
 *
 * noSuchService -
 */
-TEST(RestService, noSuchServiceAndNotFound)
+TEST(RestService, DISABLED_noSuchServiceAndNotFound)
```

Comment:
Maybe the test should be enterily removed? Related with https://github.com/apinf/fiware-orion/pull/35/review_comment/create#r208706721

Reply:
sure, we can remove all of them, not that i care very much.
however ... is it really better to remove then than to keep them to perhaps some day (soon) incorporate again?
i decided to go conservative ...

Reply:
Again, prefer to leave it.
some day we will reincorporate all these functests

So, NTC